package.json

```diff
@@ -29,7 +29,7 @@
   },
   "homepage": "https://github.com/Eppo-exp/node-server-sdk#readme",
   "dependencies": {
-    "@eppo/js-client-sdk-common": "4.8.4"
+    "@eppo/js-client-sdk-common": "4.10.0"
```

Comment:
do you want to do the minor bump for this package in this PR or follow it up before cutting the release?